test/test_export.py

```diff
@@ -472,3 +472,15 @@ def func(a, b, c):
         dynamo_result = out_graph(*flat_input)
 
         self.assertTrue(torchdynamo.utils.same(real_result, dynamo_result))
+
+    def test_min_op(self):
+        torchdynamo.reset()
+        torchdynamo.config.dynamic_shapes = True
```

Comment:
Dynamic already has a test suite. Write the test for min without this.

Reply:
I think Dynamic test suite just runs some test files under dynamic_shape = True mode. Isn't it better if I have actual test case for the min op (say inside test_misc.py). That way it is also tested under dynamic shapes.

Reply:
I believe we are saying the same thing.

Reply:
addressed